style/scalastyle.xml

```diff
@@ -119,7 +119,7 @@
  <check level="error" class="org.scalastyle.scalariform.PublicMethodsHaveTypeChecker" enabled="false"/>
  <check level="error" class="org.scalastyle.file.NewLineAtEofChecker" enabled="true"/>
  <check level="error" class="org.scalastyle.file.NoNewLineAtEofChecker" enabled="false"/>
- <check level="error" class="org.scalastyle.scalariform.BlockImportChecker" enabled="false"/>
+ <check level="error" class="org.scalastyle.scalariform.BlockImportChecker" enabled="true"/>
```

Comment:
The IDEA will place multi import in block automatically, we need to fix it manually, is there any other IDEA settings to fix it automatically or disable block import? cc @lamber-ken

Reply:
> The IDEA will place multi import in block automatically, we need to fix it manually, is there any other IDEA settings to fix it automatically or disable block import? cc @lamber-ken

Lets talk about it.